<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>F_PopUpCLESTTEM</class>
 <widget class="QWidget" name="F_PopUpCLESTTEM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lud'Open</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ImagesIcones.qrc">
    <normaloff>:/IconeLO.png</normaloff>:/IconeLO.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="GL">
     <item row="3" column="0">
      <widget class="QLabel" name="Lb_CLESTTEM">
       <property name="text">
        <string>Nom :</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="4">
      <widget class="QTableView" name="TbV_CLESTTEM">
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="4">
      <widget class="QTextEdit" name="TE_CLESTTEM"/>
     </item>
     <item row="0" column="0" colspan="4">
      <widget class="QLabel" name="Lb_TitreFenetre">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="4">
      <spacer name="VS_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="Lb_Autres">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="4">
      <spacer name="VS">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="1" colspan="3">
      <widget class="QLineEdit" name="LE_CLESTTEM"/>
     </item>
     <item row="5" column="1" colspan="3">
      <widget class="QLineEdit" name="LE_Autres"/>
     </item>
     <item row="9" column="0" colspan="4">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="Bt_Valider">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Valider</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Bt_Annuler">
         <property name="text">
          <string>Annuler</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ImagesIcones.qrc"/>
 </resources>
 <connections/>
</ui>
